src/main/java/me/contaria/seedqueue/mixin/server/MinecraftServerMixin.java

```diff
@@ -201,8 +200,7 @@ private void pauseServer(MinecraftServer server, boolean value, Operation<Void>
             return false;
         }
         if (SeedQueue.config.maxWorldGenerationPercentage < 100) {
-            WorldGenerationProgressTracker tracker = entry.getWorldGenerationProgressTracker();
-            if (tracker != null && tracker.getProgressPercentage() >= SeedQueue.config.maxWorldGenerationPercentage) {
+            if (entry.getProgressPercentage() >= SeedQueue.config.maxWorldGenerationPercentage) {

```

Comment:
You can merge these if-statements, they were only seperate because of the tracker variable